eng/templates/steps/build.yml

```diff
@@ -35,10 +35,21 @@ steps:
     git diff
   displayName: Remove RunAsPolicy From MaestroApplication
 
+- powershell: |
+    [xml]$packagesProps = Get-Content -Path .\Directory.Packages.props
+    $aspireNode = $packagesProps.Project.ItemGroup.PackageVersion | Where-Object { $_.Include -eq 'Aspire.Hosting' }
+    $fileContent = @"
+    {
+      `"microsoft.net.sdk.aspire`": `"$($aspireNode.Version)/8.0.100`"
+    }
+    "@
+    $fileContent | Out-File -FilePath $(Agent.TempDirectory)\aspire.json
+  displayName: Create Aspire Rollback File
+
```

Comment:
Should this file/step not be part of the csproj? People installing this won't be able to build the project from VS?